src/core/redux/Store.tsx

```diff
@@ -7,7 +7,8 @@ import { rootReducer } from './reducer';
 
 import { Action, State } from './types';
 
-const reduxDevToolsHook = window.__REDUX_DEVTOOLS_EXTENSION__ ? window.__REDUX_DEVTOOLS_EXTENSION__() : (x: any) => x; // tslint:disable-line no-any
+const reduxDevToolsHook =
+  process.browser && window.__REDUX_DEVTOOLS_EXTENSION__ ? window.__REDUX_DEVTOOLS_EXTENSION__() : (x: any) => x; // tslint:disable-line no-any
 const thunkMiddleware = thunk as ThunkMiddleware<State, Action>;
 
 export const initStore = (initialState?: State) =>
```

Comment:
Can't most of this file be squished into a `configureStore`?
https://redux.js.org/redux-toolkit/overview
Default devtools support and thunk middleware! 🎉

Reply:
I mixed up the PR's, this is perhaps resolved in #343?

Reply:
Yes, this is fixed in the next PR :P